marketdata/market/dex/dex.go

```diff
@@ -20,13 +20,14 @@ type Market struct {
 }
 
 func InitMarket() market.Provider {
+	update := viper.GetDuration("market.dex_quote_update_seconds")
 	m := &Market{
 		Market: market.Market{
 			Id:         "dex",
 			Name:       "Binance Dex",
 			URL:        "https://www.binance.org/",
```

Comment:
why do we pass market url here?

Reply:
I did an API to return all provider, but it's useless and I did remove